src/content/docs/quick-installation.md

```diff
@@ -30,6 +30,8 @@ The Open Data Hub operator is available for deployment in the OpenShift Operator
 1. The subscription creation view will offer a few options including *Update Channel*, keep the `rolling` channel selected. Click `Install` to deploy the opendatahub operator into the `openshift-operators` namespace.
    ![Select Channel](../assets/img/pages/docs/quick-installation/channels.png "Install")
 
+   For those who want to upgrade to new operator(version 2.X), please continue reading [here](../quick-installation-new-operator#installing-the-new-open-data-hub-operator) step 4
```

Comment:
I think we should be updating the screenshot before this step as well, that reflects the channels accurately and explicitly mention a new `fast` channel is introduced that points to a new operator

Reply:
updated the screenshot to use the one with both new and old channels available.